sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,59 @@
+* Propose, based on Jane Bennett ("Texts are bodies that can light up, by rendering human perception more acute, those bodies whose favoured vehicle of affectivity is less wordy: plants, animals, blades of grass, household objects, trash") and Mark Hansen ("all reality is mixed reality", "technics are co-constitutive of our embodied perception"), that computation is itself embodied: software has a body, and enacts real bodily effects/affects in the world
+
+
+## Community
+
+The technical portion of this project involves two connected activities:
+
+1. creating videos and music, such as _In Passing_, _Greenwich Park_, _Tofino_, and the soundtrack for _Font Màgica_, that are generated or processed by means of custom software I designed and built specifically for these pieces
+2. designing, developing, and sharing with a community a set of creative tools and frameworks, specifically _Flocking_ and _Aconite_, that were designed initially for my own creative endeavours but also intended to be used by other artists in their own creative work
+
+Every piece of "software art" in this project, along with the supporting technical frameworks and documentation, have been made using collaborative open source methods. Early ideas and concepts emerged via discussions with a distributed community of artists, designers, and programmers who have formed around the Fluid Infusion and Flocking communities<sup>1</sup>. Even this text itself, from my first tentative notes and initial annotated bibliography, was produced using open source strategies<sup>2</sup>. In the case of Flocking, a community of upwards of one hundred people, mostly artists, from around the world have used or contributed to it<sup>3</sup>.
+
+### Open as Method
+
+Open source methods first emerged within the culture of research software development in the 1970s at MIT, Berkeley, and elsewhere, and evolved into a more formalized body of practice in the late 1990s with communities such as those that develop the Linux operating system and the Firefox web browser (Voyce 416) (Bretthauer 16). In _Wide Open: Open Source Methods and Their Future_, Mulgan, Steinberg, and Salem define open source software as "any computer software distributed under a license which allows users to change or share the software's source code. Source code is the human-readable version of a computer program" (8). From my perspective, the most essential characteristic of openness as method is not so much the availability of source code (though this is a prerequisite for other forms of openness) as the way in which collaborative practices performed the context of an inclusive community have the potential to open up new creative "vectors for a collective becoming" that may not be possible with solitary, proprietary creative activity (Wark 159-60). I will discuss the importance of community and software as a vector rather than simply a set of practices below.
```

Comment:
Fascinating non-slip from the academics - Source code is *the* human-readable version